using_images/db_images/mongodb.adoc

```diff
@@ -145,65 +147,125 @@ MongoDB settings can be configured with the following environment variables.
+For example, to upload the example template into the current project's template
+library:
 
-IMPORTANT: The replication support provided by the MongoDB image is experimental and
-should not be used in production.
+====
+----
+$ oc create -f \
+    https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb_controller.json
+----
+====
 
-The full source of replication example is available here:
-https://github.com/openshift/mongodb/tree/master/2.4/examples/replica
+[IMPORTANT]
+====
+The example template does not use persistent storage. When
+you lose all members of the replication set, your data will be lost.
+====
 
-IMPORTANT: The template provided in the example above does not use persistent
-storage. When you lose all members of the replication set, your data will be
-lost.
+The following sections detail the objects defined in the example replication
+template and describe how they work together to start a cluster of MongoDB
+servers implementing master-slave replication and automated failover. This is
+the recommended replication strategy for MongoDB.
 
-This section describes how to start a cluster of MongoDB servers that implements
-master-slave replication and automated failover. This is the recommended
-replication strategy for MongoDB.
+[[creating-the-deploymentconfig]]
+=== Creating the Deployment Configuration
 
-=== Creating the DeploymentConfig
+To set up MongoDB replication, a
+link:../../architecture/core_concepts/deployments.html#deployments-and-deployment-configurations[deployment
+configuration] is defined in the example template that defines a
+link:../../architecture/core_concepts/deployments.html#replication-controllers[replication
+controller]. The replication controller manages the members of the MongoDB
+cluster.
 
-To start with MongoDB replication, you first have to define a DeploymentConfig
-that defines a replication controller which will manage the members of the
-MongoDB cluster. To tell a MongoDB server that the member will be part of the
-cluster, you have to provide additional environment variables for the container
-defined in the replication controller pod template:
+To tell a MongoDB server that the member will be part of the cluster, additional
+environment variables are provided for the container defined in the replication
+controller pod template:
 
 [cols="3a,6a,1a",options="header"]
 |===
 
-|Variable name |Description |Default
+|Variable Name |Description |Default
 
 |`*MONGODB_REPLICA_NAME*`
-|Specify the name of the replication set.
-|rs0
+|Specifies the name of the replication set.
+|*rs0*
 
 |`*MONGODB_KEYFILE_VALUE*`
-|See: http://docs.mongodb.org/manual/tutorial/generate-key-file[Generate a Key File]
-|generated
+|See: http://docs.mongodb.org/manual/tutorial/generate-key-file[Generate a Key
+File]
+|*generated*
 |===
 
-The DeploymentConfig defined in the example replication template looks like
-this:
+.Deployment Configuration Object in the Example Replication Template
```

Comment:
since your showing the json here, i'd call it a resource definition or object definition, not a resource or object.

so resource definition i guess.